docs/developers/plugins.md

```diff
@@ -2,16 +2,20 @@
 
 Plugins are the most efficient way to customize or change the default behavior of a chart. They have been introduced at [version 2.1.0](https://github.com/chartjs/Chart.js/releases/tag/2.1.0) (global plugins only) and extended at [version 2.5.0](https://github.com/chartjs/Chart.js/releases/tag/2.5.0) (per chart plugins and options).
 
-## Popular Plugins
+## Popular Chart Type Plugins
```

Comment:
I don't think `comparison.md` is a good place for this information, [notes/extensions.md](/chartjs/Chart.js/blob/master/docs/notes/extensions.md) seems a better candidate. Actually, we should also move the "Popular Plugins" into that file, then I would organize `extensions.md` as:

```markdown
# Popular Extensions
...
## Charts
...
## Plugins
...
## Integrations
### Angular
...
### React
...
```

Agree about links from `charts.md` and `plugins.md`